ida/ffxiv_structimporter.py

```diff
@@ -2,7 +2,12 @@
 # @menupath Tools.Scripts.ffxiv_structimport
 # @runtime Jython
 
-from yaml import safe_load
+from yaml import load
+try:
+    from yaml import CSafeLoader as Loader
+except ImportError:
+    from yaml import SafeLoader
```

Comment:
```suggestion
    from yaml import SafeLoader as Loader
```

This will probably fail if this isn't here

Reply:
Oooooh right. I changed that afterwards because it wasn't the Safe one!